core/src/main/java/com/google/googlejavaformat/java/Formatter.java

```diff
@@ -154,19 +160,11 @@ public CharSequence getCharContent(boolean ignoreEncodingErrors) throws IOExcept
     OpsBuilder builder = new OpsBuilder(javaInput, javaOutput);
     // Output the compilation unit.
     JavaInputAstVisitor visitor;
-    if (getMajor() >= 14) {
-      try {
-        visitor =
-            Class.forName("com.google.googlejavaformat.java.java14.Java14InputAstVisitor")
-                .asSubclass(JavaInputAstVisitor.class)
-                .getConstructor(OpsBuilder.class, int.class)
-                .newInstance(builder, options.indentationMultiplier());
-      } catch (ReflectiveOperationException e) {
-        throw new LinkageError(e.getMessage(), e);
-      }
-    } else {
-      visitor = new JavaInputAstVisitor(builder, options.indentationMultiplier());
-    }
+    if (getMajor() >= 15)
```

Comment:
Our style guide requires optional braces (https://google.github.io/styleguide/javaguide.html#s4.1-braces), e.g.:

```diff
-if (getMajor() >= 15)
+if (getMajor() >= 15) {
```

Do you mind adding those, here and below? I can take care of it when merging the PR if that's easier.

Reply:
I can fix it, but I must first ask why google-java-format doesn’t format the source like this.  I ran all sources through google-java-format 1.10.0 before committing them.  Is it intended behavior to skip braces when there’s only one statement, or is that a bug in google-java-format?

(I see after running some tests of this that braces aren’t added, but will be retained if they’re there.  So this is intended behavior?  But I still wonder why the braces aren’t added if they’re missing.)

Reply:
I just realized that I messed this up in another way.  I forgot to check for any exclusion rules in `core/pom.xml` and I see that `Java14InputAstVisitor.java` is being excluded there.  I’ll restore the code to load the visitor via reflection and make sure to set up the proper profile in `core/pom.xml` to handle `Java15InputAstVisitor.java` as well.

Reply:
> I must first ask why google-java-format doesn’t format the source like this

See https://github.com/google/google-java-format/issues/51. The formatter's job is primarily to fix issues with whitespace, there are more rules in the style guide than just whitespace

Reply:
I hadn’t realized that.  I’ll look into fixing #51 next ;-P.